<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="Udits-Air" failures="1" tests="3" name="TestNG Foundation 1" time="0.022" errors="0" timestamp="2022-04-03T12:58:20 IST">
  <testcase classname="TestNG1" name="TestCase1" time="0.001"/>
  <testcase classname="TestNG1" name="TestCase2" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at TestNG1.TestCase2(TestNG1.java:12)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- TestCase2 -->
  <testcase classname="TestNG2" name="TestCase4" time="0.001"/>
  <testcase classname="TestNG2" name="TestCase3" time="0.0">
    <ignored/>
  </testcase> <!-- TestCase3 -->
</testsuite> <!-- TestNG Foundation 1 -->
